### PR TITLE
fix: use misc to get current millis

### PR DESCRIPTION
**Pull request by hzliangbin**
ref: https://github.com/cectc/dbpack/issues/<issueID>

### CODE REVIEW

**Review by dk-lockdown — APPROVED** [review]

**Comment by fatelei** [comment]
LGTM

**Review by fatelei — APPROVED** [review]

File: pkg/misc/uuid/snowflake.go [hunk 1]
```diff
@@ -49,14 +51,14 @@ func NewWorker() (*SnowflakeWorker, error) {
 }
 
 func (w *SnowflakeWorker) NextID() (int64, error) {
-	ts := time.Now().UnixMilli()
+	ts := misc.CurrentTimeMillis()
```
Comment (bohehe) [inline pkg/misc/uuid/snowflake.go:54]:
```suggestion
	ts := misc.CurrentTimeMilliSec()
```

**Review by bohehe — APPROVED** [review]
LGTM

File: pkg/misc/uuid/snowflake.go (hunk 1 above)
Comment (dk-lockdown) [reply pkg/misc/uuid/snowflake.go:54]: there is no misc.CurrentTimeMilliSec()，suggest update at next pr

**Pull request merged**